manifest.yml.j2

```diff
@@ -57,6 +57,21 @@
       'production': 1
     },
   },
+  'notify-delivery-worker-ecs-fixup': {
+    'disk_quota': '10G',
+    'memory': '4G',
+    'additional_env_vars': {
+      'NOTIFICATION_QUEUE_PREFIX': NOTIFICATION_QUEUE_PREFIX + '-ecs',
```

Comment:
This will mean we get 2 environments with this name defined in our manifest file. I hope/assume it takes the later one (which would be this one you've defined here).